onnx/defs/schema.h

```diff
@@ -739,6 +739,19 @@ class OpSchema final {
     return all_tensor_types_ir4;
   }
 
+  static const std::vector<std::string>& all_float_types_ir4() {
```

Comment:
We don't yet have any computational op supporting float8. I think it is better if affine_grid restricts it to float16 and above.  May be we should define a set `float_16plus_types_ir4` ? Of course, we may need to check whether the function also works for all these types (in case some op doesn't support them all.)

Reply:
+1

Reply:
I think the name `all_float_types_ir4` is fine, but we should drop the float8 types from the set. We can extend it when all other ops are extended to support float8. (Otherwise, I suspect the type-checker will fail for the function-body.)

Reply:
good point. done